packages/plugin-ext/src/main/browser/tasks-main.ts

```diff
@@ -19,17 +19,40 @@ import {
     MAIN_RPC_CONTEXT,
     TaskExecutionDto,
     TasksExt,
-    TaskDto
+    TaskDto,
+    TaskPresentationOptionsDTO
 } from '../../common/plugin-api-rpc';
 import { RPCProtocol } from '../../common/rpc-protocol';
 import { Disposable, DisposableCollection } from '@theia/core/lib/common';
 import { TaskProviderRegistry, TaskResolverRegistry, TaskProvider, TaskResolver } from '@theia/task/lib/browser/task-contribution';
 import { interfaces } from 'inversify';
-import { TaskInfo, TaskExitedEvent, TaskConfiguration, TaskCustomization } from '@theia/task/lib/common/task-protocol';
+import { TaskInfo, TaskExitedEvent, TaskConfiguration, TaskCustomization, TaskOutputPresentation, RevealKind, PanelKind} from '@theia/task/lib/common/task-protocol';
 import { TaskWatcher } from '@theia/task/lib/common/task-watcher';
 import { TaskService } from '@theia/task/lib/browser/task-service';
 import { TaskDefinitionRegistry } from '@theia/task/lib/browser';
 
+const revealKindMap = new Map<number | string, string | number>(
```

Comment:
Is it better to use `number | RevealKind` here and `number | PanelKind` below?

Maybe a better notation:

```ts
const revealKindMap: Map<number, RevealKind> & Map<RevealKind, number> = new Map([
    [1, RevealKind.Always],
    [2, RevealKind.Silent],
    [3, RevealKind.Never],
    [RevealKind.Always, 1],
    [RevealKind.Silent, 2],
    [RevealKind.Never, 3]
]);
```

Reply:
Done!, Thanks !